src/spikeinterface/sorters/external/rt_sort.py

```diff
@@ -0,0 +1,199 @@
+        "max_latency_diff_sequences": "Maximum allowed weighted latency difference for sequence merging. (`float`, `2.5`)",
+        "clip_latency_diff_factor": "Latency clip = clip_latency_diff_factor * max_latency_diff. (`float`, `2`)",
+        "max_amp_median_diff_spikes": "Maximum allowed weighted percent amplitude difference for spike assignment. (`float`, `0.45`)",
+        "max_amp_median_diff_sequences": "Maximum allowed weighted percent amplitude difference for sequence merging. (`float`, `0.45`)",
+        "clip_amp_median_diff_factor": "Amplitude clip = clip_amp_median_diff_factor * max_amp_median_diff. (`float`, `2`)",
+        "max_root_amp_median_std_spikes": "Maximum allowed root amplitude standard deviation for spike assignment. (`float`, `2.5`)",
+        "max_root_amp_median_std_sequences": "Maximum allowed root amplitude standard deviation for sequence merging. (`float`, `2.5`)",
+        "repeated_detection_overlap_time": "Time window (in seconds) for overlapping repeated detections. (`float`, `0.2`)",
+        "min_seq_spikes_n": "Minimum number of spikes required for a valid sequence. (`int`, `10`)",
+        "min_seq_spikes_hz": "Minimum spike rate for a valid sequence. (`float`, `0.05`)",
+        "relocate_root_min_amp": "Minimum amplitude ratio for relocating a root electrode before first merging. (`float`, `0.8`)",
+        "relocate_root_max_latency": "Maximum latency for relocating a root electrode before first merging. (`float`, `-2`)",
+        "device": "The device for PyTorch operations ('cuda' or 'cpu'). (`str`, `cuda`)",
+        "num_processes": "Number of processes to use for parallelization. (`int`, `None`)",
+        "ignore_warnings": "Whether to suppress warnings during execution. (`bool`, `True`)",
+        "debug": "Whether to enable debugging features such as saving intermediate steps. (`bool`, `False`)",
+    }
+
+    sorter_description = """RT-Sort is a real-time spike sorting algorithm that enables the sorted detection of action potentials within 7.5ms±1.5ms (mean±STD) after the waveform trough while the recording remains ongoing.
+    It utilizes unique propagation patterns of action potentials along axons detected as high-fidelity sequential activations on adjacent electrodes, together with a convolutional neural network-based spike detection algorithm.
+    This implementation in SpikeInterface only implements RT-Sort's offline sorting.
+    For more information see https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0312438"""
+
+    installation_mesg = f"""\nTo use RTSort run:\n
+       >>> pip install git+https://github.com/braingeneers/braindance
+
+    And install the following additional dependencies:
+        >>> pip install {'torch' if os.name == 'nt' else 'torch_tensorrt'}
+        >>> pip install diptest
+        >>> pip install pynvml
+        >>> pip install scikit-learn
```

Comment:
I would recommand to make an  independant package for rt-sort with a pyproject.toml tghat would contains all the depencies.